pkg/spec3/fuzz_test.go

```diff
@@ -0,0 +1,28 @@
+package spec3
+
+import (
+	"encoding/json"
+
+	"testing"
+
+	"github.com/getkin/kin-openapi/openapi3"
```

Comment:
I could drop this test and get rid of the dependency. Why do you work so hard trying to make sure that the fuzzer generates entirely valid openapi?